README.md

```diff
@@ -48,6 +48,14 @@ Another example using fluentd with specific configuration (partial CLO configura
 
 > Note: make sure to use the sign `=` between each command-line key and value
 
+### Cluster Logging Load Client
+
+This [project](https://github.com/ViaQ/cluster-logging-load-client) is a golang application to generate logs and send them to various output destinations in various formats. The app runs as a single executable. Per configuration can spawn multiple threads. If more load is needed, scale the app horizontally. 
```

Comment:
Reformatting as follows:
The app runs as a single executable and  based on configuration it can spawn multiple threads. User can scale the app horizontally for heavy workload.